tests/integration/targets/inventory_aws_ec2/runme.sh

```diff
@@ -38,6 +38,9 @@ ansible-playbook playbooks/create_inventory_config.yml -e "template='inventory_w
 ansible-playbook playbooks/test_populating_inventory_with_constructed.yml "$@"
 ansible-playbook playbooks/create_inventory_config.yml -e "template='inventory_with_concatenation.yml.j2'" "$@"
 ansible-playbook playbooks/test_populating_inventory_with_concatenation.yml "$@"
+# generate inventory config with includes_entries_matching and prepare the tests
+ansible-playbook playbooks/create_inventory_config.yml -e "template='inventory_with_includes_entries_matching.yml.j2'" "$@"
+ansible-playbook playbooks/test_populating_inventory_with_includes_entries_matching.yml "$@"
```

Comment:
It looks like the template and playbook are missing.

I'm not sure why, but it looks like the integration tests didn't fire properly

Reply:
:thinking: indeed.